global/prometheus-alertmanager/templates/config.yaml

```diff
@@ -446,10 +555,10 @@ data:
               text: {{"'{{template \"slack.sapcc.silence1Month.actionText\" . }}'"}}
               value: {{"'{{template \"slack.sapcc.silence1Month.actionValue\" . }}'"}}
 
-    - name: slack_vmware_critical
+    - name: slack_api_critical
       slack_configs:
-        - channel: '#alert-{{"{{ .CommonLabels.tier }}"}}-{{"{{ .CommonLabels.severity }}"}}'
-          api_url: {{ required "slack.vmware_critical_webhook_url undefined" .Values.slack.vmware_critical_webhook_url | quote }}
+        - channel: '#alert-api-{{"{{ .CommonLabels.severity }}"}}'
+          api_url: {{ default "MISSING" .Values.slack.api_critical_webhook_url | quote }}
```

Comment:
Please use `{{ required "slack.api_critical_webhook_url undefined" .Values.slack.api_critical_webhook_url | quote }}`

Reply:
very good point .. I adopt for all